pkg/framework/aries/framework.go

```diff
@@ -377,24 +379,44 @@ func createVDR(frameworkOpts *Aries) error {
 	}
 
 	var opts []vdr.Option
-	for _, v := range frameworkOpts.vdr {
-		opts = append(opts, vdr.WithVDR(v))
-	}
 
 	p, err := peer.New(ctx.StorageProvider())
 	if err != nil {
 		return fmt.Errorf("create new vdr peer failed: %w", err)
 	}
 
+	// Interop: use a did:sov VDR that will prefer to store sov DID docs locally, by wrapping the did:peer VDR.
```

Comment:
@Moopli - no, you cannot override did:sov at this level. Needs to be within did exchange only.

Reply:
Let's remove this change from this PR. @Moopli